agents/plugins/smart

```diff
@@ -108,6 +108,20 @@ if which smartctl > /dev/null 2>&1 ; then
             continue
         fi
 
+        # Exclude removable media
+        if which udevadm > /dev/null 2>&1; then
+            TMP_FILE=$(mktemp)
+            udevadm info /dev/$N > $TMP_FILE
+            if grep -Fxq "E: ID_CDROM_CD=1" $TMP_FILE; then
+                rm $TMP_FILE
+                continue
+            else
+                rm $TMP_FILE
+            fi
+        else
+            [ $(< /sys/class/block/$N/removable) == "1" ] && continue
```

Comment:
I think it would be better to leave this line out, since it could also remove usb disks. I suggest that we do not try to filter if udevadm is not available.